app/scripts/constants.js

```diff
@@ -412,6 +412,14 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
       //     ]
       //   }
       // ]
+    },
+    {
+      label: "Catalog",
+      iconClass: "fa fa-book",
```

Comment:
I'd suggest the icon to be set back to pficon-service-catalog, this way, when PatternFly updates the icon, OpenShift will not need to update the code to get the new icon.  @jeff-phillips-18